README.md

```diff
@@ -94,6 +94,17 @@ Test against a specific Puppet or Facter version:
     $ PUPPET_VERSION=2.7.19 bundle update && bundle exec rake spec
     $ FACTER_VERSION=1.6.8  bundle update && bundle exec rake spec
 
+## Puppet 3 support
+
+Puppet 3 is EOL as-of January 2017. The last release of this module that will
+support Puppet 3.X and earlier will be 1.2.5
```

Comment:
Yeah, meant to edit this, was planning on bumping this version to 3.0.0 (so it matches the Puppet version 😄)

Releasing is cheap, and we've done a number of changes to the module we did since the 1.2.X version could possibly be breaking changes, so I thought might as well bump it up 👍